crypt.go

```diff
@@ -193,20 +207,64 @@ func (s *activateWithKeyDataState) run() (success bool) {
 			continue
 		}
 
-		return true
+		return true, nil
 	}
 
-	// TODO: Passphrase support
+	// Try keys that require a passphrase
+	tries := s.passphraseTries
+	var lastErr error
+
+	for tries > 0 && numPassphraseKeys > 0 {
```

Comment:
should the ActivateVolumeOptions.PassphraseTries doc be clear that tries are across keys?

Reply:
I've updated the documentation now